CONTRIBUTING.md

```diff
@@ -109,42 +115,346 @@ Check out [building instructions](https://github.com/prestodb/presto/tree/master
+
+
+## Commit Standards
+* ### Commit Size
+    * Recommended lines of code should be within +1000 lines, and should focus on one single major topic.\
+      If your commit is more than 1000 lines, consider breaking it down into multiple commits, each handling a specific small topic.
+* ### Commit Message Style
+    * **Separate subject from body with a blank line**
+    * **Subject**
+        * Limit the subject line to 10 words or 50 characters
+        * If you cannot make the subject short, you may be committing too many changes at once
+        * Capitalize the subject line
+        * Do not end the subject line with a period
+        * Use the imperative mood in the subject line
+    * **Body**
+        * Wrap the body at 72 characters
+        * Use the body to explain what and why vs. how
+        * Use the imperative mood in the body\
+          E.g. “If applied, this commit will ___________”
+        * Communicate only context (why) for the commit in the subject line
+        * Use the body for What and Why, if your commit is complex or dense, share some of the How context
+        * Assume someone may need to revert your change during an emergency
+    * **Content**
+        * **Aim for smaller commits for easier review and simpler code maintenance**
+        * All bug fixes and new features must have associated tests
+        * Commits should pass tests on their own, not be dependent on other commits\
+        * The following is recommended:
+            * Describe why a change is being made.
+            * How does it address the issue?
+            * What effects does the patch have?
+            * Do not assume the reviewer understands what the original problem was.
+            * Do not assume the code is self-evident/self-documenting.
+            * Read the commit message to see if it hints at improved code structure.
+            * The first commit line is the most important.
+            * Describe any limitations of the current code.
+            * Do not include patch set-specific comments.
+
+Details for each point and good commit message examples can be found on https://wiki.openstack.org/wiki/GitCommitMessages#Information_in_commit_messages
+
+* **Metadata**
+    * If the commit was to solve an Github issue, refer to it at the end of a commit message in a rfc822 header line format.\
+      E.g.\
+      Resolves: #1234\
+      See also: #1235, #1236\
+
+* **Backport Commits**
+    * Must refer to original commit ID (refer to the PR link in the PR message) or link and author in the commit message
+      ```bash
+      Fix OOM caused by foo in bar
+      
+      Foo was pack ratting ByteBuffers causing OOM.
+      
+      Cherry-pick of https://github.com/prestodb/presto/pull/18424/commits/30e36dae0cdb9debd991931370b86301cd17e261 
+      Co-authored-by: Foo Bar <foo@bar.com>
+      ```
+    * Try to follow the original PR/commits structure. Do not mingle multiple original commits or reorder them.
```

Comment:
```suggestion
    * Try to follow the original PR and commits structure. Do not mingle multiple original commits or reorder them.
```
Avoid slashes. See [GitLab doc recommended word list entry for slashes](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#slashes) for advice and suggestions.